lesson02/assignment/css/_components.content.scss

```diff
@@ -7,11 +7,93 @@ html {
 *:before {
     box-sizing: inherit;
 }
+@mixin margins {
+    margin: 2rem;
+}
+@mixin coloring($mycolor){
+    color: $mycolor;
+}
+body{
+    @include margins;
+
+}
 
 .main {
     padding: 2rem;
+    border-radius: 1rem;
+    background-color: $dark-gray;
+    
 }
+#content-container{
+    display: grid;
+    // grid-template-columns: repeat(auto-fit, minmax(3rem, auto));
+    grid-template-columns: 1fr 2fr;
+    border-radius: 2rem;
+    padding: 2rem;
+    background-color: $white;
+    #headshot-cell {
+        margin: 2rem;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        img {
+            border-radius: 100%;
+            width: 100%;
+        }
+        h1 {
+            @include coloring($title-text);
+        }
+        p {
+            @include coloring($title-text);
+            font-size: x-large;
+            font-style: italic;
+        }       
 
-h1 {
-    color: $red;
+    }
+    #profile-cell {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-evenly;
+        padding: 2rem;
+        height: 100%;
+        background-color: $gray;
+        
+        span {
+            @include coloring($title-text);
+        }
+        ul{
+            list-style: none;
+            padding: 0;
+            margin: 0;
+            &li {
```

Comment:
In order to vertically align the icon with the text for the LinkedIn section, you can either add a class to the last li, or use :last-child to target the last li and use flexbox.

    display: flex;
    align-items: center;
    gap: 1rem;